bigbluebutton-html5/imports/ui/components/actions-bar/mute-button/component.jsx

```diff
@@ -1,18 +1,25 @@
 import React from 'react';
 import Button from '/imports/ui/components/button/component';
+import styles from '../styles.scss';
 
 export default class MuteAudioComponent extends React.Component {
 
   render() {
-    const { isMuted, muteUser, unmuteUser } = this.props;
+    const { isInAudio, isMuted, muteUser, unmuteUser } = this.props;
     let onClick = muteUser;
     let label = 'Mute';
+    let cn = !isInAudio ? styles.invisible : null;
+    let ti = !isInAudio ? -1 : 0;
```

Comment:
Always prefer long but meaningful variable names than short ones meaningless.

`classNames` and `tabIndex` would be my suggestions.